docs/sctp.md

```diff
@@ -0,0 +1,136 @@
+# SCTP support in AKO for L4 services
+
+This feature supports SCTP protocol for L4 services in AKO. With 1.9.1, AKO will support SCTP protocol with Kubernetes/Openshift LoadBalancer services, and Gateway objects and their corresponding backend services. For more information on using Gateway class and Gateway objects with AKO, please refer to this document, https://github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/blob/master/docs/gateway-api/gateway-api.md.
+
+## Overview
+
+With version 22.1.3, AVI Controller supports SCTP traffic for L4 virtual services. To enable this feature with AKO, starting 1.9.1, AKO will support SCTP protocol in port definitions of LoadBalancer services, and Gateway objects and their corresponding backend services. Prior to 1.9.1, only TCP and UDP protocols were supported.
+
+The AVI Controller has introduced SCTP specific properties for virtual services and pools. These include an SCTP-proxy network profile for supporitng SCTP traffic in virtual services, an SCTP based health monitor for pools, and SCTP protocol match option, in L4 Policy Set match rules.
```

Comment:
It will be good to define SCTP specific property name like HM as `System-SCTP`, TCP/UPD profile`System-SCTP-proxy`, `SCTP` protocol option in L4 Policy SET

Reply:
Added specific property names.